### DISCUSSION TITLE
Is it possible to set witch display the notifications should show up on?

### DISCUSSION THREAD

**Discussion by Arash-SJPI22** [ErikReider/SwayNotificationCenter#228] [Q&A] [answered]
Is it possible to set witch display the notifications get show on if you have multiple screens and not just on the active display?
They usually get in the way of gaming and such when I don't want DND active. Would be nice to be able to set witch display they always show up on to avoid that.

**Comment by ErikReider** [comment, marked as answer]
Not currently. There's an open PR (with no code changes yet) with discussion about how to do it: https://github.com/ErikReider/SwayNotificationCenter/pull/64

Seems a bit stale so I'll look into it again :)
